src/runway/module_type.py

```diff
@@ -0,0 +1,186 @@
+"""Abstraction for the module 'type' value in a a Runway configuration."""
+# pylint: disable=unused-import
+from typing import Dict, List, Optional, Tuple, Union  # noqa: F401
+
+import glob
+import logging
+import os
+import sys
+
+from .util import (load_object_from_string)
+
+LOGGER = logging.getLogger('runway')
+
+
+# noqa pylint: disable=too-few-public-methods
+class ModuleType(object):
```

Comment:
This should _probably_ be renamed to avoid potential confusion/conflicts. It is the name of a built-in type - https://docs.python.org/3/library/types.html#types.ModuleType. (ran across this working on another project)

It doesn't matter to me what its changed to though. Maybe `RunwayModuleType` if you don't want to get creative.

Reply:
I do actually want to get creative on it, but I'm having a hard time defining it since we have "module" defined in a few places already and "type" isn't any better. Whats the feelings on ConfigType? Open to other suggestions as well, `RunwayModuleType` is a bit cumbersome but could be used in a pinch if we can't think of a better one.

Reply:
`ConfigType` may be too broad. feel free to use something different then `type` if you have a better idea for that as well. i had picked it as an easy way to infer meaning/intent in the task.

Reply:
I agree that "type" is probably the right thing here, it's just too bad these terms intermingle so much. In the interest of getting this pushed out I'm going to go with your suggestion of `RunwayModuleType` and if we think of a better name internally for it down the line we can address it at that point. Thanks for the headsup @ITProKyle